integration-tests/jest-reporter.js

```diff
@@ -32,8 +32,7 @@ module.exports = class JestReporter {
     const writableStream = createWriteStream(logFileName)
     writableStream.on('error', ({ message }) => console.error(`Error occurred while writing Console logs to ${logFileName}.\n${message}`))
 
-    testResult.console.forEach(({ message }) => writableStream.write(`${message}\n`)
```

Comment:
@slucasbroad Hi, please revert this change. `_.forEach` fails to get all console logs. Issue was fixed in https://github.com/DataBiosphere/terra-ui/pull/3024